src/CRUD.php

```diff
@@ -68,7 +68,7 @@ public function init()
     {
         parent::init();
 
-        $this->on('reload', $this->jsReload());
+        $this->stickyGet($this->paginator->name);
```

Comment:
probably same issue is with QuickSearch / Filter

Reply:
yes!